examples/04_glm_first_level/plot_design_matrix.py

```diff
@@ -114,6 +114,27 @@
 plt.show()
 
 
+# %%
+# Correlation between regressors
+# ------------------------------
+# We can plot the correlation between the regressors of our design matrix.
+# This is important to check as highly correlated regressors can affect
+# the effficieny of
+# `your design https://imaging.mrc-cbu.cam.ac.uk/imaging/DesignEfficiency#Correlation_between_regressors`_. # noqa
```

Comment:
Adding a link to this https://imaging.mrc-cbu.cam.ac.uk/imaging/DesignEfficiency#Correlation_between_regressors

I suspect things like this should probably be in the user guide too, no?